components/Layout/ThemeChanger/ThemeChanger.module.scss

```diff
@@ -0,0 +1,77 @@
+.container {
+    position: absolute;
+    top: 1rem;
+    left: -1rem;
+
+    .switch {
+        position: relative;
+        display: inline-block;
+        width: 55px;
+        height: 30px;
+
+        input {
+            /* Hide default HTML checkbox */
+            opacity: 0;
+            width: 0;
+            height: 0;
+
+            &:checked + .slider {
+                background-color: var(--color--toggle-background);
+
+                &:before {
+                    content: url('../../../public/images/smiling-sunglasses.png');
+                    -webkit-transform: translateX(26px);
+                    -ms-transform: translateX(26px);
+                    transform: translateX(26px);
+                }
+            }
+
+            &:focus + .slider {
+                box-shadow: 0 0 1px var(--color--toggle-background);
+            }
+        }
+
+        .slider {
+            position: absolute;
+            border-radius: 70px;
+            cursor: pointer;
+            top: 0;
+            left: 0;
+            right: 0;
+            bottom: 0;
+            background: var(--color--toggle-background);
+            -webkit-transition: 0.4s;
```

Comment:
**Question**

Is specifying the `-webkit-transition` needed? Will it work if it's just the `transition` below?

As far as I know, these `scss` files are usually processed in a way that `-webkit-*` are auto generated.